arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -286,5 +287,54 @@ public async Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>(string col
             }
         }
 
+        /// <summary>
+        /// Partially updates the document identified by document-handle.
+        /// The body of the request must contain a JSON document with the
+        /// attributes to patch(the patch document). All attributes from the
+        /// patch document will be added to the existing document if they do not
+        /// yet exist, and overwritten in the existing document if they do exist
+        /// there.
+        /// PATCH/_api/document/{document-handle}
+        /// </summary>
+        /// <param name="collectionName"></param>
+        /// <param name="documentKey"></param>
+        /// <param name="body"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<PatchDocumentResponse> PatchDocumentAsync(string collectionName, string documentKey, object body, PatchDocumentQuery query = null)
+        {
+            return await PatchDocumentAsync(WebUtility.UrlEncode(collectionName) + "/" + WebUtility.UrlEncode(documentKey), body, query);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="documentId"></param>
+        /// <param name="body"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<PatchDocumentResponse> PatchDocumentAsync(string documentId, object body, PatchDocumentQuery query = null)
+        {
+            ValidateDocumentId(documentId);
+            string uriString = _docApiPath + "/" + documentId;
+            if (query != null)
+            {
+                uriString += "?" + query.ToQueryString();
+            }
+            StringContent content = GetStringContent(body, true, true);
```

Comment:
We should not ignore null values nor camelCase convert this object

Reply:
The reason I say this:  users might want to set a value to `null` in the database, and they want to have control over the casing of documents they put in the database.  Also the endpoint offers a query parameter that controls special behaviour when a property is set to `null`, so we want that to be available as well.